test-utils/src/act.js

```diff
@@ -0,0 +1,12 @@
+import { options } from 'preact';
+import { setupRerender } from './setupRerender';
+
+export function act(cb) {
+	const rerender = setupRerender();
+	let flush;
+	options.afterPaint = (fc) => flush = fc;
```

Comment:
`options.afterPaint` will only be called when a component with a hook has been rendered. `flush` will be `undefined` when there is none. A check for `if (flush!=null) flush()` should be enough :+1: